modules/tinymce/src/themes/silver/main/ts/ui/dropdown/CommonDropdown.ts

```diff
@@ -158,8 +160,18 @@ const renderCommonDropdown = <T>(
 
       dropdownBehaviours: Behaviour.derive([
         ...spec.dropdownBehaviours,
-        DisablingConfigs.button(() => spec.disabled || sharedBackstage.providers.isDisabled()),
-        ReadOnly.receivingConfig(),
+        DisablingConfigs.button(() => {
+          if (prefix === MenuButtonClasses.Button) {
+            return !sharedBackstage.providers.isButtonAllowedInCurrentMode(spec.allowedModes);
+          }
```

Comment:
I can't quite work out why we need to have a specific check here. I am probably just missing something

Reply:
The `renderCommonDropdown` button is used in the menu buttons in the toolbar, and the menubar. We want the menu buttons in the menubar to stay enabled when it's in design/readonly UI enabled mode, so the menu items can be clicked.